### ISSUE TITLE
problems about inverse_folding.ipynb

### ISSUE THREAD

**Issue by winnebeauty** [ykiiiiii/GraDe_IF#10]
Hi, in the fourth block, this code--"prob,sample_graph= diffusion.ema_model.ddim_sample(input_graph)" raise a error
"too many values to unpack (expected 2)".can you please give me some help.

**Comment by ykiiiiii** [comment]
Hi @winnebeauty, I didn't find any errors when I ran the notebook. Could you please provide more information or check with the original notebook? I think the original one is good to run.

**Comment by winnebeauty** [comment]
Hi @ykiiiiii ,thanks for your reply.The error message is as follows. By the way,the original one also has the same problem.
![problem](https://github.com/ykiiiiii/GraDe_IF/assets/100328354/1b9e0176-e0e8-4772-96a3-997a63e1d39a)

**Comment by ykiiiiii** [comment]
Have you changed the neural network architecture or output? It should only have one output (AA logits). You can find what neural network output in line 151 of `gradeif.py` file. Hope it helps.
